head.html

```diff
@@ -1,6 +1,27 @@
 <meta name="viewport" content="width=device-width, initial-scale=1" />
 <script src="/blogs/scripts/lib-franklin.js" type="module"></script>
 <script src="/blogs/scripts/scripts.js" type="module"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.0/jquery.min.js"></script>
```

Comment:
I don't think we should load jQuery in the head of the page. and especially not from a third party domain.

if this is required by the header/footer service, then it should loaded by those blocks, and not globally (although ideally we could just not use it,. jQuery is a very heavy library and largely not required for what can be done with native apis in modern browsers).